pkg/operator/starter.go

```diff
@@ -249,8 +269,38 @@ func RunOperator(ctx context.Context, controllerConfig *controllercmd.Controller
 	go clusterOperatorStatus.Run(ctx, 1)
 	go logLevelController.Run(ctx, 1)
 
-	<-ctx.Done()
-	return fmt.Errorf("stopped")
+	capabilityChangedCh := make(chan struct{})
+	if !buildCapabilityEnabled {
+		// check capability periodically and close chan and return
+		go func() {
+			ticker := time.NewTicker(10 * time.Minute)
```

Comment:
This will need a doc change stating it can take up to 10 minutes to propagate the build cap enabling to the operator.